combat-trainer.lic

```diff
@@ -570,6 +576,7 @@ class LootProcess
     return false unless DRStats.necromancer?
     return false unless @ritual_type
     return false if game_state.necro_casting?
+    return true if @force_rituals
     return true if @ritual_type == 'cycle'
```

Comment:
Allows user to force rituals. Handy when farming materials or pushing hidden butcher skill.